Gaijinizer patch/Map270.txt

```diff
@@ -124,10 +138,10 @@ I and you can not "intervent" \\N[1] ...#MTLed
 文句の付け所のない、完璧なシナリオを……
 > CONTEXT: Map270/events/4/pages/0/117/Dialogue/kazuya-3
 > CONTEXT: Map270/events/6/pages/0/118/Dialogue/kazuya-3
-I prepared this "script" for you ...
-For you who want the world "no enemy" more than
-anyone...A perfect script with no complaints
-...#MTLed
+I prepared this "scenario" for you...
+It's perfect for someone like you who wishes for
+a world without enemies, surely you can't
+complain about that.#Ron
 > END STRING
```

Comment:
Some reformatting:
>_**I prepared this "scenario" for you...
It's perfect for someone like you who wishes for
a world without enemies...
Surely you can't complain about that.**_